.github/workflows/blueprint.yml

```diff
@@ -3,14 +3,38 @@ name: Compile blueprint
 on:
   push:
     branches:
-      - main # Trigger on pushes to the default branch
-  workflow_dispatch:        # Allow manual triggering of the workflow from the GitHub Actions interface
-
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+      - main
+    paths:
```

Comment:
not sure if this is really needed, also it is not capturing all relevant files in my repo, at least I see that AUTHORS.md is missing, but this is used when building the latex.

Reply:
Yes, that is a minimal set of path filters. You can add all those which are project-specific.